shared/settings/passphrase/index.desktop.js

```diff
@@ -39,28 +32,17 @@ function UpdatePassphrase (props: Props) {
         type={inputType}
         errorText={props.newPassphraseConfirmError}
         onChangeText={props.onChangeNewPassphraseConfirm}
-        style={styleInput}
-      />
+        style={styleInput} />
       <Checkbox
         label='Show typing'
         onCheck={props.onChangeShowPassphrase}
         checked={props.showTyping}
-        style={{marginBottom: globalMargins.medium}}
-      />
+        style={{marginBottom: globalMargins.medium}} />
       <Button
         type='Primary'
         label='Save'
         disabled={!props.canSave}
-        onClick={props.onSave}
-      />
-      <Text
```

Comment:
talked to marco, he was actually talking about the forgot password stuff. there is no flow for this anymore as you can change your pass whenever now